L68-core-callback-api.md

```diff
@@ -73,16 +76,31 @@ normally (corresponding to the `ok` result of
 `grpc_completion_queue_next` on the `GRPC_CQ_NEXT` queue).
 * `int inlineable`: specifies whether this functor can be run inline
 in the same thread as the one that detected it rather than sending
-it to an executor thread. This should only be
-used very carefully for trivial internally-generated callbacks that
-do not use locks.
+it to an executor thread. This must be
+used very carefully and only for trivial internally-generated callbacks that
```

Comment:
What does "internally-generated" mean here?

Reply:
Not end-user code, but rather code that is owned by the language-specific library.

Reply:
Maybe "known safe" or "library code" instead of "internally-generated", so that this doesn't imply that only callbacks from a code generator can be used here?